ops/README.md

```diff
@@ -0,0 +1,7 @@
+# Travis CI
+
+The Travis continuous integration tool is used.
+
+## Release workflow
+
```

Comment:
Don't we need to set some variables in travis so it has access to the personal docker image repository? Shouldn't you add that info in the doc under something like ``setting up your personal DIR``?

Reply:
very right, that is now corrected